### DISCUSSION TITLE
Does dropping an index from cli has a bug?

### DISCUSSION THREAD

**Discussion by nizamani** [hasura/graphql-engine#7422] [Question]
I created an index but when creating index I made a mistake and named that index wrong. When I try to delete that index from CLI I run into an issue. Issue is this:
-- I delete index Hasura CLI shows me a success message that index is deleted
-- after that I see index is still there, when I reload Hasura console index is still there
-- in migrations folder dropping index migration is created
I want to know if this is a bug if so how can I delete this index maybe through sql and later deleting that redundant migration?

Edit: Dropping index from SQL tab doesn't work either seems to be an issue with Hasura CLI when dropping an index

**Comment by motleydev** [comment]
Created an issue. #7465

**Comment by rikinsk** [comment]
@nizamani based on the discussion on the issue #7465, this might be due to a missing schema qualifier in the down statement generated for the create index statement. Can you confirm if your index was created in a schema which was not "public". sharing the contents of your up and down migration files might help with confirming this.
